prism-languages.php

```diff
@@ -4,6 +4,7 @@
  * Retrieve support languages
  * @return array Hash of lang => language
  */
+// Test PHPCS
```

Comment:
:no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).